### PR TITLE
Chk 1309 notification send

### PR DESCRIPTION
**Pull request by pietro-tota**
#### List of Changes
Add logic to handle `TransactionUserReceiptRequestedEvent` events. 
Once those events have been received a notification is build (using success/ko template based on the sendPaymentResult outcome) and sent to the user by meaning of notifications-service. 
If an error occurs communicating with notifications-service then a notification retry event in enqueued and the retry mechanism is started.

#### How Has This Been Tested?
Junit + local tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: helm/values-uat.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ microservice-chart:
       TRANSACTIONS_REFUND_RETRY_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-retry-queue
       TRANSACTIONS_REFUND_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-queue
       TRANSACTIONS_NOTIFICATIONS_RETRY_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-retry-queue
+      TRANSACTIONS_NOTIFICATIONS_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-retry-queue
```
Comment (Fernando-Granato) [inline helm/values-uat.yaml:26]: The name of the queue is wrong ?

File: helm/values-uat.yaml [hunk 2]
```diff
@@ -122,6 +123,7 @@ microservice-chart:
     TRANSACTIONS_REFUND_RETRY_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-retry-queue
     TRANSACTIONS_REFUND_QUEUE_NAME: pagopa-u-weu-ecommerce-transactions-refund-queue
     TRANSACTIONS_NOTIFICATIONS_RETRY_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-retry-queue
+    TRANSACTIONS_NOTIFICATIONS_QUEUE_NAME: pagopa-u-weu-ecommerce-transaction-notifications-retry-queue
```
Comment (Fernando-Granato) [inline helm/values-uat.yaml:126]: The name of the queue is wrong ?

File: helm/values-uat.yaml (hunk 1 above)
Comment (pietro-tota) [reply helm/values-uat.yaml]: Good catch, my bad. Fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/56/commits/c8429e16207399bd5615c548c95aa388be1ffb95

File: helm/values-uat.yaml (hunk 2 above)
Comment (pietro-tota) [reply helm/values-uat.yaml]: Good catch, my bad. Fixed with https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/56/commits/c8429e16207399bd5615c548c95aa388be1ffb95

**Review by Fernando-Granato — APPROVED** [review]

**Pull request merged**